packages/core/src/v3/runMetadata/index.ts

```
@@ -49,10 +49,30 @@ export class RunMetadataAPI implements RunMetadataManager {
   }
 
+  public incrementKey(key: string, value: number): void {
+    return this.#getManager().incrementKey(key, value);
+  }
+
+  decrementKey(key: string, value: number): void {
+    return this.#getManager().decrementKey(key, value);
+  }
+
+  appendKey(key: string, value: DeserializedJson): void {
+    return this.#getManager().appendKey(key, value);
```

Biome

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)